### PR TITLE
[Docs] Update high level documentation for each software service.

### PR DESCRIPTION
**Pull request by DanielVF**
Update high level documentation for each software service, and move software pages into main docs section for now.

I'm terribly prone to typos, so please check my speeling. :)

### CODE REVIEW

**Comment by micahalcorn** [comment]
Great, thx @DanielVF 👍 I will review.

File: docs/software/event-listener.md [hunk 1]
```diff
@@ -1,93 +1,14 @@
...
 ---
 
-# Origin Event Listener
+The Origin event listener follows the blockchain, finding origin events as they happen and passes those events on to whatever systems need this data. These events are annotated with the full information about the origin resources (listings/offers) related to the event.
 
-The Origin Event Listener follows the blockchain, looking for Origin events and passing those events on to whatever systems need that data. These events are annotated with full information about the Origin resources (listings/offers/) that fired off these events.
+The data from the listener can be used to build and keep up-to-date an offline index of all Origin Protocol data on the chain. The listener has built in support for updating an Elasticsearch and a Postgres database with the current state of listings, offers, and profiles.
 
-The data from the listener can be used to build and keep up-to-date an offline index of all Origin Protocol data on the chain.
+The listener does at-least-once notifiations. Make sure your webhooks used with the listener are idempotent, and can handle receiving the same data multiple times!
```
Comment (franckc) [inline docs/software/event-listener.md]: typo: notifications

File: docs/software/graphql-server.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+title: GraphQL Server
+nav_weight: 140
+category: Software
+---
+
+The Origin GraphQL Server provides very fast access to Origin data (both from the blockchain and from IPFS). This allows DApps to be tremendously faster than querrying the blockchain themselves and downloading and processing potentialy many distributed files per listing.
+
+Play with it live here at [graphql.originprotocol.com](https://graphql.originprotocol.com). Because it's GraphQL, there are self-documenting schemas available under the docs tab.
+
+Setup instructions can be found in the [README](https://github.com/OriginProtocol/origin/tree/master/infra/discovery/src/apollo).
```
Comment (franckc) [inline docs/software/graphql-server.md]: wrong link. should point to packagges/graphql rather than discovery.

File: docs/software/relayer.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+layout: page
+title: Relayer
+nav_weight: 80
+category: Software
+---
+
+To drive adoption, we want to be able to pay for a user's gas for certain transactions. This capability is not built into Ethereum, but can be done via contract magic. We create a proxy contract for a user and turn over ownership of that proxy to the user. The user can then sign a transaction to this proxy, which we can  submit and pay for. The proxy contract will authenicate the transaction that the user wants to run, then run it on their behalf.
```
Comment (franckc) [inline docs/software/relayer.md]: typo:authenticate

**Comment by DanielVF** [comment]
Thank you, @micahalcorn. Yeah, those sections still need a revamp, but it was time to merge this one and do those later.

**Pull request merged**